feature/platform/transceiver/tests/zr_pm_test/metadata.textproto

```diff
@@ -0,0 +1,20 @@
+# proto-file: third_party/openconfig/featureprofiles/proto/metadata.proto
+# proto-message: Metadata
+uuid: "f5d5c225-8900-42fd-8771-8614895ae036"
+plan_id: "TRANSCEIVER-6"
+description: "Telemetry: 400ZR Optics PM (Q, Pre-FEC BER, ESNR) streaming"
+testbed: TESTBED_DUT_400ZR
```

Comment:
The rundata check is failing, please run the addrundata tool from your github workspace:

cmd: `go run ./tools/addrundata --fix`

Check failure message:
Found 1 errors in feature/platform/transceiver/tests/zr_pm_test
  - metadata test description needs update: was "Telemetry: 400ZR Optics PM (Q, Pre-FEC BER, ESNR) streaming", will be "Telemetry: 400ZR Optics pm streaming."
F0308 15:4[9](https://github.com/openconfig/featureprofiles/actions/runs/8197673092/job/22443145162?pr=2779#step:3:10):25.304436    3064 main.go:[10](https://github.com/openconfig/featureprofiles/actions/runs/8197673092/job/22443145162?pr=2779#step:3:11)0] Rundata check found problems.  Please run: go run ./tools/addrundata --fix

Reply:
Done